addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -441,6 +441,7 @@ registry.slider = publicWidget.Widget.extend({
         this._computeHeights();
         // Initialize carousel and pause if in edit mode.
         this.$target.carousel(this.editableMode ? 'pause' : undefined);
+        $(window).on('resize.slider', () => this._computeHeights());
```

Comment:
You can never listen to scroll and resize events without throttling/debouncing the related function. Here debounce